docs/requirements.txt

```diff
@@ -7,3 +7,5 @@ Sphinx==3.*
 sphinx-github-changelog
 sphinx-autodoc-typehints
 sphinxcontrib-programoutput
+sqlalchemy>=1.4,<2.0.0
+django>=2.2
```

Comment:
Can we rather change `.` at the top of the file to `.[django,sqlalchemy]` ?

Reply:
Me, Galaxy brain, makes a comment requesting a change, then merges immediately.

Reply:
Good point. I'll create a PR for that if you haven't done it already.